Tctl admin historyhost (#1147)
temporalio/documentation: docs/tctl/admin/history_host/get_shardid.md

---
id: get_shardid
title: tctl admin history_host get_shardid
description: Providing the shardId on command
tags:
  - reference
  - tctl
---

> The `tctl admin history_host get_shardid` command gets the shardId for a given namespaceId and workflowId.

Reviewer: ```suggestion
The `tctl admin history_host get_shardid` command gets the `shardId` for a given `namespaceId` and `workflowId`.
```

Alias: `gsh`

## Modifiers

> The modifiers listed below change the behavior of this command.

Reviewer: ```suggestion
The following modifiers change the behavior of this command.
```

#### `--namespace_id value`

> The namespaceId of the history host where we're getting the shardId.

Reviewer: ```suggestion
The `namespaceId` of the history host where we're getting the `shardId`.
```
